KIPs/kip-draft_kgp6.md

```diff
@@ -0,0 +1,734 @@
+---
+kip: 103
+title: Treasury Fund Rebalancing
+author: Aidan (@aidan-kwon), Toniya (@toniya-klaytn)
+discussions-to: https://govforum.klaytn.foundation/t/kgp-6-proposal-to-establish-a-sustainable-and-verifiable-klay-token-economy/157
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-02-24
+---
+
+## Simple Summary
+
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the KIP.-->
+
+This proposal suggests a smart contract interface standard that records the rebalance of treasury funds. The main objective is to facilitate the approval and redistribution of treasury funds to new addresses while keeping record of the Retired and Newbie details.
+
+## Abstract
+
+<!--A short (~200 word) description of the technical issue being addressed.-->
+
+Organizations need to manage treasury funds in a transparent and accountable manner. The proposed standard aims to make the management of treasury funds more transparent by recording the rebalance of treasury funds. The smart contract will keep records of the addresses which hold the treasury funds before and after rebalancing. It also facilates approval before execution.
```

Comment:
Could you please write when and how the records in the contract are used?
Is this just for record purposes? Without the transfer of KLAY?

Reply:
Yes this is to only for record purpose